compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -871,6 +872,45 @@ public void visit(BLangSimpleVariable varNode) {
         transferForkFlag(varNode);
     }
 
+    private void analyzeModuleConfigurableAmbiguity(BLangPackage pkgNode) {
+        if (pkgNode.moduleContextDataHolder == null) {
+            return;
+        }
+        ModuleDescriptor rootModule = pkgNode.moduleContextDataHolder.descriptor();
+        Map<String, PackageID> configKeys = new HashMap<>();
+        Set<BVarSymbol> configVars = symResolver.getConfigVarSymbols(pkgNode.symbol);
+        populateModuleKeys(configVars, rootModule, configKeys);
```

Comment:
Can't `configKeys` be created in `populateModuleKeys` and returned now.

Let's change the name also to `getModuleKeys` or `getModuleConfigKeys` if we are changing this.